daemonize.py

```diff
@@ -68,7 +68,10 @@ def exit(self):
         Cleanup pid file at exit.
         """
         self.logger.warn("Stopping daemon.")
-        os.remove(self.pid)
+        try:
+            os.remove(filename)
+        except FileNotFoundError:
```

Comment:
`FileNotFoundError` exception was introduced in Python 3.3, see PEP 3151, so it's broken almost always.